### PR TITLE
Completed javascript-light-bulb challenge.

### PR DESCRIPTION
**Pull request by AnthonyVNgo**

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Anthony! Please attach a gif of your functionality in action, then send me a message on Slack once this has been updated.

**Comment by AnthonyVNgo** [comment]
Kapture 2022-03-29 at 21 21 53

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+var bgOff;
+
+// console.log(buttonOn);
+// console.log(bgOn);
+
+var button = 0;
+
+function buttonSwitch(event) {
+
+  if (button === 0) {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js]: I would suggest using a boolean rather than `0` and `1`

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    buttonOn.className = 'button-off column-25 height-25 align-self-center border-radius-circle';
+    bgOn.className = 'bg-off';
+    button++;
+    // console.log(button);
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js]: Please remove all code comments prior to submitting for review.

File: javascript-light-bulb/main.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+    buttonOff = document.querySelector('div.button-off');
+    bgOff = document.querySelector('.bg-off');
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:16]: This is unnecessary. Even if you change classes on the element you selected, the variables holding a DOM element will hold the same element. You can just make one variable to hold the `bulb` element and one variable to hold the `background` element.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**